src/rest-server/src/templates/yarnContainerScript.mustache

```diff
@@ -57,11 +43,13 @@ PS4="+[\t] "
 trap exit_handler EXIT
 
 
-mkdir -p "/tmp/pai-root/alive/$APP_ID"
+alive_dir="tmp/pai-root/alive"
```

Comment:
>alive_dir [](start = 0, length = 9)

should we link this folder to a yarn folder so Yarn can clean it up?

Reply:
YARN wil not delete symlink folders.